src/main/java/org/rest/config/ETagConfiguration.java

```diff
@@ -0,0 +1,13 @@
+package org.rest.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.filter.ShallowEtagHeaderFilter;
+
+@Configuration
+public class ETagConfiguration {
+    @Bean
+    public ShallowEtagHeaderFilter shallowEtagHeaderFilter() {
```

Comment:
Does it work now for all the requests? If we want to specify some URLs that need to be catchable we need to wrap your Filter into FilterRegistrationBean, right?

Reply:
Correct